docs/setup/sample-runs.md

```diff
@@ -112,20 +74,23 @@ systemctl enable containerd.service
 
 ## Docker Compose
 
-[Docker Compose](https://docs.docker.com/compose/) is included with Docker, and allows you to deploy a self-contained network.
+[Docker Compose](https://docs.docker.com/compose/) is included with Docker,
+and allows you to deploy a self-contained network.
 We're using it to simplify deploying the Cumulus ETL project in your ecosystem.
 
-The docker-compose.yaml, which defines the network, adds the following containers:
+The `compose.yaml` file, which defines the network, adds the following containers:
 
 - The Cumulus ETL process itself
 - A [cTAKES](https://ctakes.apache.org/) server, to handle natural language 
   processing of clinical notes.
+- A [cNLP transformer](https://github.com/Machine-Learning-for-Medical-Language/cnlp_transformers)
+  server, to run a second-pass negation of cTAKES results.
 
 ## Cumulus ETL
 
 ### Building a Docker Image
-Again, because Cumulus isn't yet a public project, we don't offer a turnkey docker image.
-But we _do_ ship a Dockerfile that you can use to build a docker image yourself for now.
+We plan to eventually offer a turnkey docker image.
+But for now you have to build a docker image yourself.
```

Comment:
Once we get a docker image, these instructions will get a lot prettier all over again.